.idea/misc.xml

```diff
@@ -58,5 +59,5 @@
   <component name="PDMPlugin">
     <option name="skipTestSources" value="false" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_21" default="true" project-jdk-name="temurin-21" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_X" default="true" project-jdk-name="temurin-21" project-jdk-type="JavaSDK" />
```

Comment:
There's expectations for the developer's IDE described here:
https://github.com/kroxylicious/kroxylicious/blob/main/DEV_GUIDE.md#intellij

Reply:
I'll revert this change, it was not intentional